### ISSUE TITLE
Add fade when teleporting

### ISSUE THREAD

**Issue by egingric** [RinglingGameArt/2014-Racing-Game#638]
Can't figure out how to do this. It is 1:30 in the AM so brain = muddled.

**Comment by rhuberga** [comment]
Please also disable ResetVehicle remote event after RaceOver.

**Comment by egingric** [comment]
Does it reset the vehicle, or just pop up a warning? That would be a different bug so please make a new bug for that.

**Comment by rhuberga** [comment]
What I was trying to say was, just add this to the kismet map:
![untitled-1](https://cloud.githubusercontent.com/assets/7010882/2670134/d6dce92a-c0c9-11e3-8191-acbe0622c685.jpg)

I should stop trying to use words when screenshots would be more effective.

**Comment by egingric** [comment]
Right. Thanks! Can you create a bug for this please? This bug is a
different bug from the one we are conversing in.

Regards,
-Eric
